README.md

```diff
@@ -141,38 +151,60 @@ $sparky = new SparkPost($httpClient, ['key'=>'YOUR_API_KEY']);
 ### Send An Email Using The Transmissions Endpoint
 ```php
 <?php
+require 'vendor/autoload.php';
+
 use SparkPost\SparkPost;
 use GuzzleHttp\Client;
 use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
 
 $httpClient = new GuzzleAdapter(new Client());
 $sparky = new SparkPost($httpClient, ['key'=>'YOUR_API_KEY']);
 
+$sparky = new SparkPost($httpClient, $options);
 $promise = $sparky->transmissions->post([
     'content' => [
-        'from'=> [
-            'name' => 'Sparkpost Team',
-            'email' => 'from@sparkpostbox.com'
+        'from' => [
+            'name' => 'SparkPost Team',
+            'email' => 'from@sparkpostbox.com',
         ],
-        'subject'=>'First Mailing From PHP',
-        'html'=>'<html><body><h1>Congratulations, {{name}}!</h1><p>You just sent your very first mailing!</p></body></html>',
-        'text'=>'Congratulations, {{name}}!! You just sent your very first mailing!'
+        'subject' => 'First Mailing From PHP',
+        'html' => '<html><body><h1>Congratulations, {{name}}!</h1><p>You just sent your very first mailing!</p></body></html>',
+        'text' => 'Congratulations, {{name}}!! You just sent your very first mailing!',
     ],
-    'substitution_data'=> ['name'=>'YOUR_FIRST_NAME'],
-    'recipients'=> [
-        [ 'address' => '<YOUR_EMAIL_ADDRESS>' ]
+    'substitution_data' => ['name' => 'YOUR_FIRST_NAME'],
+    'recipients' => [
+        [
+            'address' => [
+                'name' => 'YOUR_NAME',
+                'email' => 'YOUR_EMAIL',
+            ],
+        ],
+    ],
+    'cc' => [
+        [
+            'address' => [
+                'name' => 'ANOTHER_NAME',
+                'email' => 'ANOTHER_EMAIL',
+            ],
+        ],
+    ],
+    'bcc' => [
+        [
+            'address' => [
+                'name' => 'AND_ANOTHER_NAME',
+                'email' => 'AND_ANOTHER_EMAIL',
+            ],
+        ],
     ],
-    'bcc' =>  [
-        ['address' => '<ANOTHER_EMAIL_ADDRESS>' ]
-    ]
 ]);
-?>
 ```
 
 ### Send An API Call Using The Base Request Function
 We may not wrap every resource available in the SparkPost Client Library, for example the PHP Client Library does not wrap the Metrics resource. To allow you to use the full power of our API we created the `request` function which allows you to access the unwrapped resources.
```

Comment:
I think this needs to be re-written to stop talking about wrapped/unwrapped. It's just "we provide a base request to make any calls.." or something like that.